### PR TITLE
uar-711 add bo/mo validation

### PR DESCRIPTION
**Pull request by AndreKramer7654**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-711

### Change description
Put back the Bo/Mo validation. No validation against the statement made.

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by jkayyala — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java [hunk 1]
```diff
@@ -155,9 +160,7 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
                 entityDtoValidator.validate(entityDto, errors, loggingContext);
             }
         }
-        // Temporarily disabling BO/MO validation till it is implemented in Update Journey UAR-711
```
Comment (AndreKramer7654) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java:158]: The comment needs removing but not line 159.

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java [hunk 2]
```diff
@@ -71,6 +71,11 @@ public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionD
     }
 
     private void validateFullUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        // Method to be added to as Update journey developed
+        validateFullCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficers(overseasEntitySubmissionDto, errors, loggingContext);
```
Comment (AndreKramer7654) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java:77]: There is another validateOwnersAndOfficers call in the partial validation that should be enabled.

**Comment by AndreKramer7654** [comment]
Converted to draft as this PR is about to be superceeded.

**Comment by AndreKramer7654** [comment]
Work done in https://github.com/companieshouse/overseas-entities-api/pull/312

**Pull request closed without merging**